packages/webdriverio/src/scripts/customElement.ts

```diff
@@ -17,13 +17,13 @@ export default function customElementWrapper () {
                 parentNode = parentNode.parentNode
             }
             console.debug('[WDIO]', 'newShadowRoot', this, parentNode, parentNode === document)
-            return origConnectedCallback.call(this)
+            return origConnectedCallback && origConnectedCallback.call(this)
```

Comment:
let's use modern js syntax
```suggestion
            return origConnectedCallback.call?.(this)
```

Reply:
I'm fine with using the `?` operator, but it would have to be after origConnectedCallback in that case:
`return origConnectedCallback?.call(this)`